content/blog/2021-04-15-april-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,192 @@
+﻿---
+title: April '21 Heartbeat
```

Comment:
Merged, but I see that we should also change the file name ^^

and images like `/uploads/images/2021-04-15/made-with-ml-logo.png` - I'm not even sure why it does work now